shell/platform/android/io/flutter/embedding/engine/dart/DartExecutor.java

```diff
@@ -121,7 +121,10 @@ public void executeDartEntrypoint(@NonNull DartEntrypoint dartEntrypoint) {
     Log.v(TAG, "Executing Dart entrypoint: " + dartEntrypoint);
 
     flutterJNI.runBundleAndSnapshotFromLibrary(
-        dartEntrypoint.pathToBundle, dartEntrypoint.dartEntrypointFunctionName, null, assetManager);
```

Comment:
The null here was just a feature mysteriously and annoyingly missing on Android for some reason https://github.com/flutter/flutter/issues/72630.